### DISCUSSION TITLE
importing input file

### DISCUSSION THREAD

**Discussion by sdpatil07** [GreenleafLab/ArchR#1800] [Questions / Documentation] [answered]
Hello,
I was going through the quick tutorial and wanted to follow it. But i want to use data that is already available in my lab. As i understood in a quick tutorial you have used 
inputFiles <- [getTutorialData](https://www.archrproject.com/reference/getTutorialData.html)("Hematopoiesis") to get the data into R. 

 Could you please tell me, which function i should use instead? i read the full user manual and there you mentioned ArchR uses “scanTabix” to read fragment files and “scanBam” to read BAM files. As i could understand the quick tutorial you have used fragments.tsv.gz as input files is it correct?

Thanks for your help. Sorry if this question sounds foolish but i am new to R and i am trying to learn these things on my own. 

Dont need to answer anything, i explored Archr git hub files more and found what i was looking for. Sorry for the inconvenience.  I tried to delete this but couldn't

**Comment by rcorces** [comment, marked as answer]
The full manual discusses how to create Arrow files using the `createArrowFiles()` function both for fragment files and for bam files
